website/docs/upgrade-guides.md

```diff
@@ -6,6 +6,87 @@ sidebar_label: Upgrade Guides
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
+## @ionic/portals 0.6.x -> 0.7.0
+
+IonicPortals iOS and Android version 0.7.0 is compatible with '@ionic/portals' version 0.7.x
+
+:::caution
+Ionic Portals 0.7.0 is a notable update that upgrades the Capacitor dependency to version 4. Care should be taken to update dependencies across your web content and native apps to ensure compatibility.
+:::
+
+First review the [Capacitor 4 Update Guide](https://capacitorjs.com/docs/updating/4-0) for an overview of necessary changes. Some will not be relevant for Portals apps, but this will be a useful reference in case you encounter issues with your upgrade.
+
```

Comment:
We could make note of the update to the `InitialContext` change. The previous definition of it was:
```typescript
export interface InitialContext<T> {
  name: string,
  value: T
}
```
Now it's
```typescript
export interface InitialContext<T> {
  name: string,
  value: T | undefined
}
```

This was to always make InitialContext available in a Portals context regardless of the native side passing in any specific data or not so that the web side could have access to it's name for useful things like `publish(`${portalName}:dismiss`)`